code/modules/atmospherics/portable/portable_pressurizer.dm

```diff
@@ -0,0 +1,405 @@
+#define FAN_OFF 0
+#define FAN_ON_INLET 1
+#define FAN_ON_OUTLET 2
+#define PROCESS_OFF 0
+#define PROCESS_ACTIVE 1
+#define PROCESS_PAUSED 2
+#define MIN_BLAST_DELAY (5 SECONDS)
+#define MAX_BLAST_DELAY (30 SECONDS)
+#define BLAST_EFFECT_RATIO (0.7)
+
+/obj/machinery/portable_atmospherics/pressurizer
```

Comment:
would like a doc comment explaining what this is

Reply:
Added